src/GraphQL/Types/Scalars/EnumerationGraphType.cs

```diff
@@ -78,25 +80,25 @@ public void AddValue(EnumValueDefinition value)
         };
 
         /// <inheritdoc/>
-        public override object ParseValue(object value) => value switch
+        public override object? ParseValue(object? value) => value switch
         {
             string s => Values.FindByName(s)?.Value ?? ThrowValueConversionError(value),
             null => null,
             _ => ThrowValueConversionError(value)
         };
 
         /// <inheritdoc/>
-        public override bool CanParseValue(object value) => value switch
+        public override bool CanParseValue(object? value) => value switch
         {
             string s => Values.FindByName(s) != null,
             null => true,
             _ => false
         };
 
         /// <inheritdoc/>
-        public override object Serialize(object value)
+        public override object? Serialize(object? value)
         {
-            if (value == null)
+            if (value == null) // TODO: why? null as internal value may be mapped to some external enumeration name
```

Comment:
@Shane32 Pay attention to this, please.

Reply:
Because in general, enumerations are value types and null would be a nullable enum. If null was able to be mapped, it would break the normal use case and throw an exception because null is unable to be matched. We should not support mapping null for enums as a built-in feature. People can override Serialize and the other methods if they want to serialize to/from null.